Update the contributing section docs (#3221)
facebook/react-native-website: website/contributing/release-stable-minor.md

---
id: release-stable-minor
title: Minor Stable 0.Y.0
---

:::info
Documents in this section go over steps to run different types of React Native release updates. Its intended audience is those in [relevant release roles](./release-roles-responsibilites.md).
:::

### Pre-requisites

- Write access to [react-native](https://github.com/facebook/react-native) repository.
- Write access to [releases](https://github.com/reactwg/react-native-releases) repository.
- One CircleCI personal API token - see [here](https://circleci.com/docs/2.0/managing-api-tokens/#creating-a-personal-api-token) how to set one.

> - Have the blog post ready as a PR .
> - Blogpost PR should be ready to merge (for `react-native-website` [repository](https://github.com/facebook/react-native-website)).

Reviewer: ```suggestion
- Blog post PR should be ready to merge (for `react-native-website` [repository](https://github.com/facebook/react-native-website)).
```

Small cleanup and naming unification.

- Documentation PR should be ready to merge (for `react-native-website` [repository](https://github.com/facebook/react-native-website)).
- Changelog PR should be ready to merge (for `react-native` [repository](https://github.com/facebook/react-native)).
- Previous RC has been thoroughly tested and no important issues have been reported. When releasing stable minor, you **should not** include any new cherry-picks.

### 1. Publish the release

```bash
# Make sure you are on the release branch of the version
git checkout -b 0.Y-stable

./scripts/bump-oss-version.js --to-version 0.Y.0 --token <YOUR_CIRCLE_CI_TOKEN>
> Do you want this to be latest?
# Reply to this prompt with "yes".
# This updates npm registry to point to this version as "latest"
```

> When this is done, all the other PRs (changelog, documentation, blogpost) should also get merged.

Reviewer: ```suggestion
When this is done, all the other PRs (changelog, documentation, blog post) should also get merged.
```

You can find the whole changelog history in the [changelog.md file](https://github.com/facebook/react-native/blob/main/CHANGELOG.md).
```

> It's likely that when you post this GitHub release, the PRs for changelog, blogpost and documentation are still in the rollout phase. You can start publishing the release anyway (to do step #3) with placeholders instead of links - but make sure to come back and update them once everything is out!

Reviewer: ```suggestion
It's likely that when you post this GitHub release, the PRs for changelog, blog post and documentation are still in the rollout phase. You can start publishing the release anyway (to do step #3) with placeholders instead of links - but make sure to come back and update them once everything is out!
```
